articles/quickstart/webapp/any/01-login.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Login
+default: true
+description: This tutorial demonstrates how to integrate Auth0 authentication into any conventional server-based web application.
+budicon: 
+---
+
+This guide explains in general terms how to integrate Auth0 authentication into a conventional server-based web app serving static HTML pages to the user. 
+
```

Comment:
It can serve any kind of page, not only static HTML ones

Reply:
Right.